applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_transient_thermal_element.cpp

```diff
@@ -500,7 +500,8 @@ KRATOS_TEST_CASE_IN_SUITE(CheckElement_Throws_When3DPropertyHasInvalidValue, Kra
     const ProcessInfo& r_current_process_info = model_part.GetProcessInfo();
     KRATOS_EXPECT_EXCEPTION_IS_THROWN(
         p_element->Check(r_current_process_info),
-        "THERMAL_CONDUCTIVITY_SOLID_ZZ has an invalid value at element 1")
+        "HERMAL_CONDUCTIVITY_SOLID_ZZ in the properties with Id 0 at element with Id 1 has an "
```

Comment:
It seems like the starting 'T' has been left out accidentally:
```suggestion
        "THERMAL_CONDUCTIVITY_SOLID_ZZ in the properties with Id 0 at element with Id 1 has an "
```

Reply:
Wow, how was it working? Fixed.